src/Groups/homomorphisms.jl

```diff
@@ -616,6 +616,26 @@ function isomorphism(::Type{GrpAbFinGen}, A::GrpAbFinGen)
    end::AbstractAlgebra.Generic.IdentityMap{GrpAbFinGen}
 end
 
+# We need not find independent generators in order to create
+# a presentation of a fin. gen. abelian group.
+function isomorphism(::Type{FPGroup}, A::GrpAbFinGen)
+   # Known isomorphisms are cached in the attribute `:isomorphisms`.
+   isos = get_attribute!(Dict{Type, Any}, A, :isomorphisms)
```

Comment:
Maybe this for type stability?
```suggestion
   isos = get_attribute!(Dict{Type, Any}, A, :isomorphisms)::Dict{Type, Any}
```

Reply:
Good idea (also in the other instances of this code line)